pytorch_lightning/trainer/trainer.py

```diff
@@ -1390,25 +1392,45 @@ def __set_ckpt_path(self, ckpt_path: Optional[str], model_provided: bool, model_
         if ft_checkpoints:
             ft_ckpt_path = ft_checkpoints[0].ckpt_path
             fs = get_filesystem(ft_ckpt_path)
-            if fs.exists(ft_ckpt_path):
-                return ft_ckpt_path
+            if not fs.exists(ft_ckpt_path):
+                ft_ckpt_path = None
+        else:
+            ft_ckpt_path = None
```

Comment:
this can all be simplified by setting `ft_ckpt_path = None` from the beginning

Reply:
Done